src/core/packages/chrome/browser-internal/src/ui/project/sidenav_v2/navigation/to_navigation_items.tsx

```diff
@@ -85,11 +85,17 @@ export const toNavigationItems = (
     );
   }
 
+  if (!logoNode) {
+    warnOnce(
+      'Navigation tree is missing a logo node. The first level should contain a logo node with solution logo, name and home page href.'
+    );
+  }
+
   const logoItem: SideNavLogo = {
-    href: warnIfMissing(logoNode, 'href', '/') as string,
+    href: warnIfMissing(logoNode, 'href', '/missing-href-😭'),
     iconType: warnIfMissing(logoNode, 'icon', 'logoKibana') as string,
```

Comment:
Ahhh, fair, I must've copied the line but `as string` should be applied to `iconType` because it should expect a narrowed type. `href`, `id` and `label` are just strings. Great catch 👌🏻